art/urls.py

```diff
@@ -18,19 +18,32 @@
 from django.conf.urls import include
 from django.conf.urls.static import static
 from django.urls import path
-from rest_framework_swagger.views import get_swagger_view
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
 
 from api import urls
 
-schema_view = get_swagger_view(title='ART API')
+schema_view = get_schema_view(
```

Comment:
Also let's be consistent with the string quotes. Noticed usage of both double and single quotes. Let's refactor the file to use single quotes.

Reply:
@alexkiura done